addons/payment/models/chart_template.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+
+
+class WizardMultiChartsAccounts(models.TransientModel):
+    _inherit = 'wizard.multi.charts.accounts'
+
+    @api.multi
+    def execute(self):
+        res = super(WizardMultiChartsAccounts, self).execute()
+
+        # Search for installed acquirers modules
+        acquirer_modules = self.env['ir.module.module'].search(
+            [('name', 'like', 'payment_%'), ('state', '=', 'installed')])
+        acquirer_names = [a.name.split('_')[1] for a in acquirer_modules]
+
+        # Search for acquirers having no journal
+        acquirers = self.env['payment.acquirer'].search(
+            [('provider', 'in', acquirer_names), ('journal_id', '=', False), ('company_id', '=', self.company_id.id)])
+
+        # Try to generate the missing journals
+        acquirers.try_loading_for_acquirer()
+        return res
```

Comment:
This method is supposed to return dictionaries representing `account.bank.accounts.wizard` records... I don't see the point in my case..

Reply:
This method is supposed to return dictionaries representing account.bank.accounts.wizard records... I don't see the point in my case..

Reply:
because it's giving default values for the method _create_bank_journals_from_o2m()... but you could directly inherit that method as well.

Don't be afraid to do a little refactoring: my point is that what you want to achieve in this method is barely the same as what _create_bank_journals_from_o2m() is doing, so that where you should add the acquirer journals creation. I also think each payment acquirer module should add its own journal only.